Sources/SwiftSyntaxBuilderGeneration/Tokens.swift.gyb

```diff
@@ -0,0 +1,135 @@
+    super.init(name: name, kind: "pound_\(kind)", serializationCode: serializationCode, unprefixedKind: kind, text: text, classification: classification, isKeyword: true, requiresTrailingSpace: true)
+  }
+}
+
+public class PoundObjectLiteral: PoundKeyword {
+  public let description: String
+  public let `protocol`: String
+
+  public init(name: String, kind: String, serializationCode: Int, text: String, classification: String = "ObjectLiteral", description: String, `protocol`: String) {
+    self.description = `description`
+    self.`protocol` = `protocol`
+    super.init(name: name, kind: kind, serializationCode: serializationCode, text: text, classification: classification)
+  }
+}
+
+public class PoundConfig: PoundKeyword { }
+
+public class PoundDirectiveKeyword: PoundKeyword {
+  public override init(name: String, kind: String, serializationCode: Int, text: String, classification: String = "PoundDirectiveKeyword") {
+    super.init(name: name, kind: kind, serializationCode: serializationCode, text: text, classification: classification)
+  }
+}
+
+public class PoundConditionalDirectiveKeyword: PoundDirectiveKeyword {
+  public override init(name: String, kind: String, serializationCode: Int, text: String, classification: String = "PoundDirectiveKeyword") {
+    super.init(name: name, kind: kind, serializationCode: serializationCode, text: text, classification: classification)
+  }
+}
+
+public class Punctuator: Token { }
+
+public class Literal: Token { }
+
+public class Misc: Token { }
+
+let SYNTAX_TOKENS: [Token] = [
+% for token in SYNTAX_TOKENS:
+%   class_name = type(token).__name__
+%   text = 'text: "' + token.text + '", ' if token.text else ""
+%   if class_name in ['Keyword', 'SwiftKeyword', 'DeclKeyword', 'StmtKeyword', 'ExprKeyword', 'PatternKeyword', 'SilKeyword']:
+    ${class_name}(name: "${token.name}", serializationCode: ${token.serialization_code}, text: "${token.text}"),
+%   elif class_name in ['PoundKeyword', 'PoundConfig', 'PoundDirectiveKeyword', 'PoundConditionalDirectiveKeyword']:
+    ${class_name}(name: "${token.name}", kind: "${token.kind}", serializationCode: ${token.serialization_code}, text: "${token.text}"),
+%   elif class_name in ['Punctuator', 'Misc']:
+    ${class_name}(name: "${token.name}", kind: "${token.kind}", serializationCode: ${token.serialization_code}, ${text}requiresLeadingSpace: ${str(token.requires_leading_space).lower()}, requiresTrailingSpace: ${str(token.requires_trailing_space).lower()}),
+%   elif class_name == 'Literal':
+    ${class_name}(name: "${token.name}", kind: "${token.kind}", serializationCode: ${token.serialization_code}),
+%   elif class_name == 'PoundObjectLiteral':
+    ${class_name}(name: "${token.name}", kind: "${token.kind}", serializationCode: ${token.serialization_code}, ${text}description: "${token.description}", protocol: "${token.protocol}"),
+%   else:
+    fatalError("Unknown token `${class_name}`")
```

Comment:
I don’t think `fatalError` is the right thing to do here, because you would insert `fatalError("Unknown token…")` into the generated source code. What you want is probably closer to 
```suggestion
%     fatal_error("Unknown token `${class_name}`")
```

with 
```python
def printerr(message):
    print(message, file=sys.stderr)


def fatal_error(message):
    printerr(message)
    sys.exit(1)
```

taken from build-script.py. If this is the only place where we need a Python `fatal_error`, we can also just inline the function bodies here.